css/src/atomics/border.scss

```diff
@@ -0,0 +1,85 @@
+$borders-directions: (
```

Comment:
Think that perhaps we should just move `directions` to tokens. It seems like we could encounter this type of usage in multiple places.

Reply:
Updated! Also updated `spacing.scss` file to use that object as well!